### PR TITLE
Introduce TSLint

### PR DESCRIPTION
**Pull request by kenju**
Introduced rules:

- tslint:recommended (https://github.com/palantir/tslint/blob/master/src/configs/recommended.ts)
- tslint-react (https://github.com/palantir/tslint-react)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,17 +20,10 @@ jobs:
...
-
-      - run:
-          name: Run Test
-          command: npm run test:coverage
-
-      - run:
-          name: Report Coverage
-          command: npm run coveralls
+      - run: npm install --no-emoji --no-progress
+      - run: npm run lint
```
Comment (kenju) [inline .circleci/config.yml:24]: run `npm run lint` on ci as well

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "extends": [
+    "tslint:recommended",
```
Comment (kenju) [inline tslint.json:3]: basically follow the `tslint:recommended` options, but ignoring some rules being unrelated with the code quality itself.

ignoring rules examples:

- alphabetic order
- format (parenthesis, spaces, etc)
- alignment

**Comment by kenju** [comment]
@bitjourney/kibela-developers 

Please review this PR 🙏

**Comment by gfx** [comment]
LGTM

**Comment by kenju** [comment]
Thank you!

**Pull request merged**